projects/plugins/jetpack/class.jetpack.php

```diff
@@ -5573,178 +5164,6 @@ public static function maybe_set_version_option() {
 
 	/* Client Server API */
 
-	/**
-	 * Loads the Jetpack XML-RPC client.
-	 * No longer necessary, as the XML-RPC client will be automagically loaded.
-	 *
-	 * @deprecated since 7.7.0
-	 */
-	public static function load_xml_rpc_client() {
```

Comment:
This function is called by the following plugins in the w.org repo:

- [ ] `akismet`: class.akismet-admin.php,1136
- [ ] `certly-guard`: class.certly-admin.php,948
- [ ] `jetpack-subscription-form`: jetpack-subscription.php,253
- [ ] `vaultpress`: vaultpress.php,3004 and vaultpress.php,3023

Reply:
All good:
- `akismet`: Wrapped in `if ( defined( 'JETPACK__VERSION' ) && version_compare( JETPACK__VERSION, '7.7', '<' )  ) {` ([#](https://plugins.trac.wordpress.org/browser/akismet/tags/4.1.10/class.akismet-admin.php#L1136))
- `vaultpress`: Both instances wrapped in `if ( defined( 'JETPACK__VERSION' ) && version_compare( JETPACK__VERSION, '7.7', '<' ) && ! class_exists( 'Jetpack_IXR_Client' ) ) {` ([#](https://plugins.trac.wordpress.org/browser/vaultpress/trunk/vaultpress.php#L3003))

Would break things:

- `jetpack-subscription-form` This plugin "adds customizable UI" to our subscription form. Last updated 5 years ago, 500+ active installs. It calls this function without any type of check (would fatal if active with Jetpack deactivated). https://wordpress.org/plugins/jetpack-subscription-form/

Would break, but don't care :)
- `certly-guard` -- I didn't check this one. It has under 10 active installs and requires an API Key from a defunct site..